From: Vedran Miletić <vedran@miletic.net>
Date: Mon, 20 Jun 2016 23:17:49 +0000 (+0200)
Subject: clover: Fix build against clang SVN >= r273191
X-Git-Url: http://repo.or.cz/mesa.git/commitdiff_plain/82e0bbd01abfb2073519941d9893fa6ac05fb58c?hp=cd89c834a8b3b4e5f5874c8e1f90c9b01d541181

clover: Fix build against clang SVN >= r273191

setLangDefaults() now requires PreprocessorOptions as an argument.

Reviewed-and-Tested-by: Michel Dänzer <michel.daenzer@amd.com>
---

diff --git a/src/gallium/state_trackers/clover/llvm/invocation.cpp b/src/gallium/state_trackers/clover/llvm/invocation.cpp
index e2cadda..57eaaaa 100644
--- a/src/gallium/state_trackers/clover/llvm/invocation.cpp
+++ b/src/gallium/state_trackers/clover/llvm/invocation.cpp
@@ -207,7 +207,7 @@ namespace {
       c.getDiagnosticOpts().ShowCarets = false;
       c.getInvocation().setLangDefaults(c.getLangOpts(), clang::IK_OpenCL,
 #if HAVE_LLVM >= 0x0309
-                                        llvm::Triple(triple),
+                                        llvm::Triple(triple), c.getPreprocessorOpts(),
 #endif
                                         clang::LangStandard::lang_opencl11);
       c.createDiagnostics(
